### ISSUE TITLE
Incident Response - Review and document how the team and leadership can escalate during an outage 

### ISSUE THREAD

**Issue by jhouse-solvd** [department-of-veterans-affairs/va.gov-team#20689]
## Description
_What details are necessary for understanding the specific work or request tracked by this issue?_

## Background/context/resources

- https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Postmortems/2021-02-11-VA.gov%20Site%20Outage.md
- https://dsva.slack.com/archives/CJRQ85PQB/p1614181773027900
- https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/19843

## Technical notes
_Notes around work that is happening, if applicable_

---
## Tasks
- [ ] _What things need to happen?_

## Definition of Done
- [ ] _What is the expected outcome of this issue?_

---
### Reminders
- [X] Please attach your team label and any other appropriate label(s) 
- [X] Please attach the needs grooming tag if needed
- [X] Please connect to an epic

**Comment by drorva** [comment]
Let's tackle this first thing in Q2.

**Comment by jhouse-solvd** [comment]
@drorva - Do you recommend that we promote this to an initiative? If so, we would love your help in defining some of the criteria to make it successful.

**Comment by drorva** [comment]
Yes please. I believe @f1337 has done quite a bit of work on this already.

**Comment by dginther** [comment]
@dginther to look into this, there has been some recent work done on this, and there may be a duplicate ticket, or another ticket that will tie into this.
